src/installers.ts

```diff
@@ -58,7 +65,14 @@ const MOD_FILE_EXT = ".archive";
  * The extension of a RedScript file
  */
 const REDSCRIPT_FILE_EXT = ".reds";
-
+/**
+ * The extension of a JSON file
+ */
+const JSON_FILE_EXT = ".json";
+const KNOWN_JSON_FILES = {
```

Comment:
Are these game files, or from a known mod? In the latter case if you want to specifically account for these, I think a test would be a good place — can leave the logic here as generic as it can be but also ensure it does the right thing.

Reply:
They are part of the game/